src/main/scala/se/nimsa/dicom/streams/ModifyFlow.scala

```diff
@@ -118,16 +118,10 @@ object ModifyFlow {
       def findModifyPart(header: HeaderPart): List[DicomPart] = sortedModifications
         .find(m => m.matches(tagPath))
         .map { tagModification =>
-          if (header.length > 0) {
-            currentHeader = Some(header)
-            currentModification = Some(tagModification)
-            value = ByteString.empty
-            Nil
-          } else {
-            val newValue = tagModification.modification(ByteString.empty)
-            val newHeader = header.withUpdatedLength(newValue.length)
-            newHeader :: valueOrNot(newValue)
-          }
+          currentHeader = Some(header)
+          currentModification = Some(tagModification)
+          value = ByteString.empty
+          Nil
```

Comment:
The above is an unrelated fix that slipped in. Else-part of this if-else cannot happen.